packages/common/src/ro/planningSession.ro.ts

```diff
@@ -9,12 +38,33 @@ export class PlanningSessionRO {
   profileOption?: string;
 
   @Expose()
-  careLocationId?: string;
+  careSetting!: PlanningSessionCareSettingRO;
 
   @Expose()
   updatedAt!: Date;
 
+  @Expose()
+  bundles!: PlanningSessionBundleRO[];
+
   constructor(data: any) {
     Object.assign(this, data);
+
+    /** Bundles */
+    const bundles: PlanningSessionBundleRO[] = [];
+    if (data?.careActivity?.length > 0) {
+      data.careActivity.forEach((ca: any) => {
```

Comment:
Can you add a type here?

Reply:
No - Learned from experience that importing Entities to "common" from "api" leads to RunTime errors. It doesn't translate to @tbcm/common instead the complete non-resolvable path.